build.gradle

```diff
@@ -13,16 +13,24 @@ buildscript {
     }
     mavenCentral()
   }
+  repositories {
+    maven {
+      url "https://plugins.gradle.org/m2/"
+    }
+  }
```

Comment:
Is this canonical for Gradle to repeat the `repositories` block twice? It seems strange, but I'm not a Gradle expert.